src/ResourceManager/PowerBIEmbedded/AzureRM.PowerBIEmbedded.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '4.0.0'
+ModuleVersion = '5.0.0'
```

Comment:
@selasagi please revert this version bump - we will handle bumping the versions during the release

Reply:
done